CHANGES.md

```diff
@@ -16,6 +16,8 @@ next
   current directory. This allows Emacs and Vim to know where relative
   filenames should be interpreted from. Fixes #138
 
+- Add support for locks in actions, for flaky tests
+
```

Comment:
hey, my tests are not flacky :p they just happen to use an OS resources (e.g. a port) and they would like to have exclusive access to it :-)

Reply:
I see, that makes sense